.github/workflows/test.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       - name: Run Server migrations
         env:
           NODE_ENV: test
+          DATABASE_URL: postgres://latitude:secret@localhost:5432/latitude_test
```

Comment:
When I changed to `db` the `packages/env/test.env` the CI started failing. Would be ideal to call `db` the postgresql in CI but I didn't know how. So I override the env here